src/audio/detect_test.c

```diff
@@ -169,10 +169,15 @@ static void default_detect_test(struct comp_dev *dev,
 	uint32_t sample;
 	uint16_t valid_bits = cd->sample_valid_bytes * 8;
 	const int32_t activation_threshold = cd->config.activation_threshold;
+	uint32_t cycles_per_frame; /**< Clock cycles required per frame */
```

Comment:
Typo in commit message: "Then only a fraction" or just "For a single frame, only a fraction"

Reply:
ack